tests/integration/helpers.py

```diff
@@ -5,10 +5,13 @@
 
 import json
 import logging
+import pathlib
```

Comment:
Another place where we should just import `Path` directly? Looks like it's just used for the `Awaitable` annotation in the `build_and_deploy_charm` function annotation.

Reply:
Good catch!